feature-libs/asm/components/customer-list/customer-list.model.ts

```diff
@@ -0,0 +1,7 @@
+import { CustomerListColumnActionType } from '@spartacus/asm/root';
+import { User } from '@spartacus/core';
+
+export interface CustomerListActionEvent {
```

Comment:
Can we rename to it `CustomerListAction`? We have `CxEvent`, this interface is not Spartacus' event.